src/test/java/com/jalasoft/schoology/steps/EventSteps.java

```diff
@@ -22,7 +23,7 @@ public EventSteps(Dashboard dashboard, EventDetails eventDetails, EventForm even
         this.eventForm = eventForm;
     }
 
-    @When("I click the \"Events\" link")
+    @And("I click the \"Events\" link")
```

Comment:
when you use "" in Gerkin is to send  one parameter :
And I validate course "planningTest1" is listed in Courses
In this case you can just send: And I click the Events link.  without quotations marks. Please let me know your thoughts.